### DISCUSSION TITLE
I have a 120fps webcam, why cant neuralnet be set to120fps?

### DISCUSSION THREAD

**Discussion by SsXxX87** [opentrack/opentrack#1968] [General]
I have a 120fps webcam, why cant neuralnet be set to120fps?

only 30 and 60 fps options are available, is their a way to unlock it to 120fps?

**Comment by sthalik** [comment]
It's not really practical due to CPU usage.

**Reply by renzhefeilong** [reply]
Actually, I also want to feedback this problem, my CPU is AMD R5 5600G, and the 120 fps camera can reach more than 100 fps with single threading.
